.travis.yml

```diff
@@ -24,7 +25,7 @@ before_script:
 
 
 script:
-  - pytest --cov=app
+  - coverage run --source=app/my_api/tests/v2 -m pytest && coverage report
```

Comment:
This points at testing the tests folder, consider pointing to the app version 2 folder where your app is at.